BuildConfigurations/StripeiOS-Shared.xcconfig

```diff
@@ -30,12 +30,14 @@ CLANG_WARN_ASSIGN_ENUM = YES
 CLANG_WARN_DOCUMENTATION_COMMENTS = YES
 CLANG_WARN_UNGUARDED_AVAILABILITY = YES
 
+
 // Code Signing Identity
 // 
 // The name ("common name") of a valid code-signing certificate in a keychain within your
 // keychain path.   A missing or invalid certificate will cause a build error.
 
-CODE_SIGN_IDENTITY[sdk=iphoneos*] =
+CODE_SIGN_STYLE = Automatic
+DEVELOPMENT_TEAM = Y28TH9SHX7
```

Comment:
I don't think we want this — it'll make building painful for anyone not in our development team.

Reply:
yeah, I was debating which way to go with this, on the one hand this will make it painful for anyone not in our development team, but if we don't add it, every time we want to run on device we will have to set the development team in the project, which would be reset every time the project is generated.
I guess we could load this values from an environment variable or something like that too

Reply:
I reverted this to what it was before and now I'm setting the values at generation time based on environment variables `TUIST_CODE_SIGN_IDENTITY` and `TUIST_DEVELOPMENT_TEAM`, this way we can just have the team add those variables, which I will add to the instructions.

Reply:
SGTM!